core/src/main/resources/application.yml

```diff
@@ -77,12 +70,30 @@ feast:
       bootstrapServers: localhost:9092
       replicationFactor: 1
       partitions: 1
-
     specsOptions:
       specsTopic: feast-specs
       specsAckTopic: feast-specs-ack
       notifyIntervalMilliseconds: 1000
+      
+  security:
+    authentication:
+      enabled: false
+      provider: jwt
+    authorization:
+      enabled: false
+      provider: none
```

Comment:
We should implements some sensible defaults here for `authorization` section with the `keto` authorization provider. The rule of the thumb is that user has to change the least amount of config for `Keto` auth to work.

Reply:
Yea that makes sense, but what does this tangibly mean? That we will deploy a Keto instance? If not, then the default configuration won't work.